extension/content_script.js

```diff
@@ -620,6 +620,8 @@ function removeUI() {
 function toggleExtensionVisibility() {
 	// Doc is not readable.
 	if (doc.getNumSentences() === 0) {
+		// Remove load icon
+		removeLoadIcon();
```

Comment:
on non readable sites the gear shows up forever

Reply:
Good catch